test/integration/api/event.spec.js

```diff
@@ -118,6 +118,32 @@ test('get events history with date filter', async (t) => {
   })
 })
 
+test('fails to get events history with same date boundary type filters', async (t) => {
```

Comment:
Maybe `redundant relational operators` is more explicit.

Same for your commit message `Same boundary in range filter…` -> `Redundant relational operators`